src/Servers/HttpSys/src/RequestProcessing/RequestStream.cs

```diff
@@ -77,10 +77,12 @@ public override long Seek(long offset, SeekOrigin origin)
 
     public override void SetLength(long value) => throw new NotSupportedException(Resources.Exception_NoSeek);
 
-    public override void Flush() => throw new InvalidOperationException(Resources.Exception_ReadOnlyStream);
```

Comment:
Was this the only consumer of `Resources.Exception_ReadOnlyStream`?

Reply:
Good catch, it does appear to be unused after this change.

Reply:
Unresolving, in case you were using it to track work.

Reply:
Thanks - turns out I was wrong, it's still used here:

https://github.com/dotnet/aspnetcore/blob/43daa49193408a047e1df3fe1fc44dea80c4b8ca/src/Servers/HttpSys/src/RequestProcessing/RequestStream.cs#L302-L315